### ISSUE TITLE
Cannot run Valama due to missing shared library

### ISSUE THREAD

**Issue by Gilnaa** [Valama/valama#101]
While running Valama from the terminal:

``` bash
gilnaa@gilnaa-Lenovo-G550:~$ valama -d
valama: error while loading shared libraries: libguanako.so.1: cannot open shared object file: No such file or directory 
```

**Comment by albel1205** [comment]
I still cannot start the valama :(

`````````````````````````````````````````
[kientnt90@localhost:0 build]$ valama -d
valama: error while loading shared libraries: libguanako.so.1: cannot open shared object file: No such file or directory
```````````````````````````````````

**Comment by LSenf** [comment]
How did you install and run valama?

**Comment by albel1205** [comment]
I tried to install valama in Fedora, and the sudo make install just worked fine.
When I run the valama from terminal,  I got the error message as below


`````````````````````````````````````````
valama: error while loading shared libraries: libguanako.so.1: cannot open shared object file: No such file or directory

`````````````````````````````````````````

**Comment by inizan-yannick** [comment]
may be you have missed "sudo ldconfig" command

**Comment by albel1205** [comment]
I already ran the "sudo ldconfig" command, but It didn't work. Then I tried to figure out the reason, and I found that the path to libguanako.so.1 library wasn't included in the /etc/ld.so.conf file. Therefore, I have to manually add it to the ld.so.conf, and run the "sudo ldconfig" again.

I am able to start the valama ide for now. Just want to provide some information for people who are in a similar case. 

Thanks for all of your supports.
